test/ui-2/test/ColorPicker.test.js

```diff
@@ -7,19 +7,29 @@ beforeAll(async () => {
 
 describe('Color Picker', () => {
   test('Color Picker is not breaking other input fields', async () => {
+    // Wait for .step to load
     await page.waitForSelector('.step');
+    // Click and select crop step
     await page.click('[data-value=\'crop\']');
+    //Check to see of color picker input is present
```

Comment:
Please add a space at the start of the comment. (to make all the comments look good and readable)

Reply:
Done!!